tests/test_base_interface.py

```diff
@@ -23,7 +21,20 @@ def pfbi(pf_app):
 
 @pytest.fixture
 def activate_test_project(pfbi):
-    pfbi.app.ActivateProject(PF_PROJECT_PATH)
+    """The project for testing must be located in the current
+    user under "powfacpy\\powfacpy_tests". This method will create
+    a copy of that project which is then used for the tests. This 
+    ensures that the tests are always run with the same initial
+    project state.
+    """
+    user = pfbi.app.GetCurrentUser()
+    project_for_testing = pfbi.get_single_obj(r"powfacpy\powfacpy_tests",
+        parent_folder = user)
+    folder_of_project_for_testing = pfbi.get_single_obj(r"powfacpy",
+        parent_folder = user)    
+    project_copy = pfbi.copy_single_obj(project_for_testing,
+        folder_of_project_for_testing,new_name="powfacpy_tests_copy_where_tests_are_run")    
+    project_copy.Activate()    
```

Comment:
is the copy also deleted after the tests have run?

Reply:
I thought about that but it should not be deleted because you want to see the effects of the tests on the project (for debugging for example)